packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -22,29 +24,45 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
       [onChange]
     );
     const styles = useStyles2(getCheckboxStyles);
+    const { inputStyle, checkmarkStyle, labelStyle, descriptionStyle } = getPropertiesForCheckboxSize(styles, size);
 
     return (
       <label className={cx(styles.wrapper, className)}>
         <input
           type="checkbox"
-          className={styles.input}
+          className={inputStyle}
           checked={value}
           disabled={disabled}
           onChange={handleOnChange}
           {...inputProps}
           ref={ref}
         />
-        <span className={styles.checkmark} />
-        {label && <span className={styles.label}>{label}</span>}
-        {description && <span className={styles.description}>{description}</span>}
+        <span className={checkmarkStyle} />
+        {label && <span className={labelStyle}>{label}</span>}
+        {description && <span className={descriptionStyle}>{description}</span>}
       </label>
     );
   }
 );
 
+interface CheckboxStyles {
+  wrapper: string;
+  smInput: string;
+  smCheckmark: string;
+  smLabel: string;
+  smDescription: string;
+  mdInput: string;
+  mdCheckmark: string;
+  mdLabel: string;
+  mdDescription: string;
+  lgInput: string;
+  lgCheckmark: string;
+  lgLabel: string;
+  lgDescription: string;
+}
```

Comment:
You do not need these, you can remove checkbox from getFormStyles it's not used anywhere